docs/cent-node/overview/introduction.mdx

```diff
@@ -7,12 +7,15 @@ redirect_from:
   - /cent-node/
 ---
 
+<!-- Apply katex css -->
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.11.0/dist/katex.min.css" integrity="sha384-BdGj8xC2eZkQaxoQ8nSLefg4AV4/AwB3Fj+8SUSo7pnKP6Eoy18liIKTPn9oBYNG" crossOrigin="anonymous"/>
```

Comment:
Can we not have this in the mdx but have it be injected automatically outside of the content?

Reply:
Also, I'd prefer not to load this from a CDN but package it into our existing JS bundle.